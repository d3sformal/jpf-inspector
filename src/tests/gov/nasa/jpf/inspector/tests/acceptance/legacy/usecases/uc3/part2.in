cr bp mi=*Predecessor2:callMePred2
run
print #heap[356]
cr bp #heap[356].pred2_i1 > 45
set #heap[356].pred2_i1 = 50
print #heap[356]
print #heap[356].#static
run
thread_pc
set #heap[356].pred2_i1 = 20
run
wait