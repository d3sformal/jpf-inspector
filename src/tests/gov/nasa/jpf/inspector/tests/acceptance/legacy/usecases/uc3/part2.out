INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 method_invoke=*Predecessor2:callMePred2]
#heap[356] (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc3.Predecessor3) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  Instance fields
	0 : pred1_i1 (int) = 101
	1 : pred1_i2 (int) = 102
	2 : pred1_i3 (int) = 103
	3 : pred1_i4 (int) = 104
	4 : a (int) = 1
	5 : a (int) = 2
	6 : pred2_i1 (int) = 201
	7 : pred2_i2 (int) = 202
	8 : pred2_i3 (int) = 203
	9 : pred2_i4 (int) = 204
	10 : a (int) = 3
	11 : pred3_i1 (int) = 301
	12 : pred3_i2 (int) = 302
	13 : pred3_i3 (int) = 303
	14 : pred3_i4 (int) = 304
  Static fields
	0 : pred3_i11 (int) = 311
	1 : pred3_i12 (int) = 312
	2 : pred3_i13 (int) = 313
	3 : pred3_i14 (int) = 314
Value set successfully.
#heap[356] (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc3.Predecessor3) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  Instance fields
	0 : pred1_i1 (int) = 101
	1 : pred1_i2 (int) = 102
	2 : pred1_i3 (int) = 103
	3 : pred1_i4 (int) = 104
	4 : a (int) = 1
	5 : a (int) = 2
	6 : pred2_i1 (int) = 50
	7 : pred2_i2 (int) = 202
	8 : pred2_i3 (int) = 203
	9 : pred2_i4 (int) = 204
	10 : a (int) = 3
	11 : pred3_i1 (int) = 301
	12 : pred3_i2 (int) = 302
	13 : pred3_i3 (int) = 303
	14 : pred3_i4 (int) = 304
  Static fields
	0 : pred3_i11 (int) = 311
	1 : pred3_i12 (int) = 312
	2 : pred3_i13 (int) = 313
	3 : pred3_i14 (int) = 314
#heap[356].Predecessor3 (gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc3.Predecessor3) = gov.nasa.jpf.inspector.tests.acceptance.legacy....
  No instance fields
  Static fields
	0 : pred3_i11 (int) = 311
	1 : pred3_i12 (int) = 312
	2 : pred3_i13 (int) = 313
	3 : pred3_i14 (int) = 314
INFO: Breakpoint hit: [2 :  state=ENABLED hits=1 hitsTotal=1 (#heap[356].pred2_i1 > 45)]
INFO: Breakpoint hit: [2 :  state=ENABLED hits=2 hitsTotal=2 (#heap[356].pred2_i1 > 45)]
0 : gov/nasa/jpf/inspector/tests/acceptance/legacy/usecases/uc3/Print02.java:95:    System.out.println("# Predecessor2.callMePred2(" + a + ")");
	gov.nasa.jpf.inspector.tests.acceptance.legacy.usecases.uc3.Predecessor2:callMePred2:0:getstatic java.lang.System.out
Value set successfully.
INFO: JPF Terminating